pcsx2/R5900.cpp

```diff
@@ -440,7 +440,7 @@ __fi void _cpuEventTest_Shared()
 
 	if ((psxHu32(HW_ICFG) & (1 << 3))) // if were in PS1 mode adjust ee accordingly.
 	{
-		cpuSetNextEventDelta((static_cast<float>((g_iopNextEventCycle - psxRegs.cycle))*8.71) - EEsCycle);
+		cpuSetNextEventDelta((static_cast<int>((g_iopNextEventCycle - psxRegs.cycle)*8.71)) - EEsCycle);
```

Comment:
You need an else statement here to put the one in above that you didn't touch. No point in doing it twice.